.github/workflows/test.yml

```
@@ -55,16 +48,16 @@ jobs:
             sudo apt-get install -y libsoxr-dev
         shell: bash
 
+      - name: Install llvm # required for rune
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)